rust/main/agents/relayer/src/msg/op_submitter.rs

```diff
@@ -105,7 +107,7 @@ impl SerialSubmitter {
     pub fn new(
         domain: HyperlaneDomain,
         rx: mpsc::UnboundedReceiver<QueueOperation>,
-        retry_op_transmitter: Sender<MatchingList>,
+        retry_op_transmitter: &Sender<Arc<MessageRetryRequest>>,
```

Comment:
`Sender` holds an `Arc` internally as well btw https://docs.rs/tokio/latest/src/tokio/sync/broadcast.rs.html#165

don't think a reference to an Arc would save that much memory - unless you think it's more ergonomic or clearer, would keep this arg without a ref